README.md

```diff
@@ -79,7 +79,7 @@ You can install from source using Go modules:
 2. Then use the install command:
 
    ```bash
-   go install github.com/openfga/openfga/cmd/openfga
```

Comment:
Wait, what's wrong with this? This `go install` the package and puts it in `$GOBIN`. I'd prefer we leave this as is as it has no extra dependencies required on GNU Make.

Reply:
Sorry, I have updated the wrong place.